contents/tutorials/churn-rate.md

```diff
@@ -0,0 +1,87 @@
+---
+title: How to calculate and lower churn rate with PostHog
+sidebar: Docs
+showTitle: true
+author: ['ian-vanagas']
+date: 2022-09-28
+featuredImage: ../images/tutorials/banners/actions.png
+topics: ['trends']
+---
+
+- **Level:** Easy 🦔
+- **Estimated reading time:** 6 minutes ☕️
+
+The word churn once reminded people of fresh butter, now it brings thoughts of lost users and revenue to product and marketing teams.
+
+The churn rate is the percentage of users who stop an action during a certain period. It is used to understand the negative changes in usage, revenue, or other growth metrics. Understanding churn rate is key to figuring out problem areas in your product. 
+
+In this tutorial, we’ll calculate and visualize the churn rate then use PostHog’s features of session recordings, cohorts, and actions to lower it.
+
+### Prerequisites
+
+To follow this tutorial along, you need to have:
+
+1. **[Deployed PostHog](https://posthog.com/docs/deployment)**.
```

Comment:
```suggestion
### Prerequisites

To follow this tutorial along, you need to have:

1. **[Deployed PostHog](https://posthog.com/docs/deployment)**.
```

I feel like we can drop this section. I wouldn't consider it mandatory for all tutorials.